packages/server/graphql/queries/suOrgCount.ts

```diff
@@ -35,29 +35,41 @@ export default {
     requireSU(authToken)
 
     // RESOLUTION
-    return r
-      .table('Organization')
-      .getAll(tier, {index: 'tier'})
-      .merge((organization) => ({
-        users: (r
-          .table('OrganizationUser')
-          .getAll(organization('id'), {index: 'orgId'})
-          .filter({removedAt: null})
-          .eqJoin('userId', r.table('User'))
-          .zip() as any)
-          .filter((user) =>
-            r.branch(
-              r(ignoreEmailRegex).eq(''),
-              true,
-              user('email')
-                .match(ignoreEmailRegex)
-                .eq(null)
-            )
-          )
-          .filter((user) => r.branch(includeInactive, true, user('inactive').not()))
-          .count()
-      }))
-      .filter((org) => org('users').ge(minOrgSize))
+    if (!ignoreEmailRegex && !includeInactive)
+      return (r
+        .table('OrganizationUser')
+        .getAll(
+          ([tier, false] as unknown) as string, // super hacky type fix bc no fn overload is defined in the type file for this valid invocation
```

Comment:
You're right @tiffanyhan, this is _super_ hacky. I think we should try and do better. 

@mattkrick what would you suggest here?

Reply:
@mattkrick do you have any suggestion for a better way to get around the type error here?

Reply:
resolving convo as there were no suggestions